Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -351,10 +351,13 @@ struct ContentData {
         Normal,
         None,
         String,
+        Image,
     } type { Type::Normal };
 
     // FIXME: Data is a list of identifiers, strings and image values.
     String data {};
+    Optional<RefPtr<AbstractImageStyleValue const>> image {};
```

Comment:
Just `RefPtr` (not `Optional<RefPtr)`) is fine, because it either holds a value or null.